backend/coverage/clover.xml

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I saw your comment about ignoring the files maybe we should delete this generated folder and add it to .gitignore.
Then we can just screen shot the coverage stats to put in the readme?